app/models/waste_carriers_engine/transient_registration.rb

```diff
@@ -85,6 +85,7 @@ def pending_payment?
     end
 
     def pending_manual_conviction_check?
+      return false unless metaData.ACTIVE?
```

Comment:
Is this right? Sorry, head full of cold and a day covering childcare but isn't it `return false unless metaData.PENDING?`?

If it was ACTIVE wouldn't it have already been approved? Unless `metaData.ACTIVE` comes from the original registration and we do nothing with it, in which case this completely makes sense 😁

Reply:
It's the latter! When we copy data to the transient registration, that includes the metaData.status. So if the original reg was active, the transient reg will also be active.